.github/workflows/ci.yml

```diff
@@ -70,26 +64,9 @@ jobs:
         path: node_modules
         key: node_modules3
     - run: yarn install
-    - run: |
-        node script/entry.js domVisitor errorsExist \
-          --bookId="$BOOK_ID" \
-          --bookVersion="$BOOK_VERSION" \
-          --rootUrl "$BASE_URL" \
-          --queryString="validateLinks"
+    - run: bash ./script/validate-modified-books.bash "$BASE_URL"
```

Comment:
pretty sure you don't need `bash` on the front of this. and i think it would be fine to use `BASE_URL` from the environment instead of a numbered param inside the script